projects/aca-shared/rules/src/app.rules.ts

```diff
@@ -624,3 +625,19 @@ export function canOpenWithOffice(context: AcaRuleContext): boolean {
 
   return context.permissions.check(file, ['update']);
 }
+
+function isSmartFolder(context: RuleContext): boolean {
+  if (context.selection && !context.selection.isEmpty) {
+    const node = context.selection.first;
+    if (!node?.entry.isFolder) {
+      return false;
+    }
+    const nodeAspects = getNodeAspectNames(node);
+    return nodeAspects.includes('smf:customConfigSmartFolder') || nodeAspects.includes('smf:systemConfigSmartFolder');
+  }
+  return false;
+}
+
+function getNodeAspectNames(node: any): any[] {
```

Comment:
aspectNames is list of strings, so return type of that function should be string[]

<img width="447" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/3a560b61-9c92-4690-b081-85589382729d">

Reply:
@AleksanderSklorz thanks for CR!